src/cc2olx/tools/video_upload.py

```diff
@@ -123,6 +127,41 @@ def make_generate_upload_link_request(url, data, filename, access_token):
     return response
 
 
+def upload_transcript(filename, edx_video_id, language_code, access_token):
+    """
+    Make a POST request against the Studio upload transcript API and return the
+    response. If errors occur during the API call, log to the console.
+
+    Arguments:
+        * filename: the transcript filename
+        * edx_video_id: the video ID of the video this transcript is for
+        * language_code: the language of the transcript
```

Comment:
Please add access token to docstring args

Reply:
Thanks for catching that. It's added now.